src/core.js

```diff
@@ -1041,6 +1041,8 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
   }
 
   function validateChanges(changes, source, callback) {
+    verifyChanges();
```

Comment:
Do you think that separated synchronous function is necessary? Maybe we can achieve the same effect using a body of `verifyChanges()` function directly in `validateChanges`. What's your opinion of my idea?